src/resources/Pipelines/StatementGroups/StatementGroupsInterfaces.ts

```diff
@@ -52,6 +53,25 @@ export interface ListStatementGroupsOptions {
      * This allows you to search within group name, group user notes and condition definition.
      */
     filter?: string;
+
+    /**
+     * The list of group status allowed in the results.
+     * If you leave this parameter <em>undefined</em> or pass an empty array, all group's status will be allowed in the results.
+     */
+    status?: Array<StatementGroupsStatusFilter>;
+
+    /**
+     * The list of group types allowed in the results.
+     * If you leave this parameter <em>undefined</em> or pass an empty array, all group types will be allowed in the results.
+     */
+    types?: Array<StatementGroupType>;
+}
+
+export enum StatementGroupsStatusFilter {
```

Comment:
Move to [Enums.ts](https://github.com/coveo/platform-client/blob/master/src/resources/Enums.ts)? IIRC otherwise you won't be able to import the enum properly outside the project. It might have been fixed since the last time I encountered this problem though.